drivers/src/data_vault.rs

```diff
@@ -390,6 +390,25 @@ impl DataVault {
         self.read_cold_reset_entry4(ColdResetEntry4::LmsVendorPubKeyIndex)
     }
 
+    /// Set the rom cold boot status.
+    ///
+    /// # Arguments
+    ///
+    /// * `status` - Rom Cold Boot Status
+    ///
+    pub fn set_rom_cold_boot_status(&mut self, status: u32) {
```

Comment:
Maybe the name should make it clear is this also locking the entry? Or perhaps get rid of this function and call write_lock_cold_reset_entry4 directly?

Reply:
This is inline with all other functions on the data vault driver. I have updated the comment on the function; if you feel strongly, we can change the naming for all these functions to indicate that the entry is also being locked, as a separate change.